src/engine/BMGame.php

```diff
@@ -912,7 +912,11 @@ protected function set_swing_values() {
             if (!$player->waitingOnAction) {
                 // check for unique swing values
                 if (!self::are_unique_swing_values_valid($player)) {
-                    $this->message = 'The swing values chosen are not unique';
+                    if ('Gordo' == $player->button->name) {
```

Comment:
Hmm, wait... i'm not going to update the dev site for now, because i'm confused about how this fix is relevant, and i think we can talk about it on this issue just as easily.  Does this have anything to do with whether the button is Gordo?  I thought the problem was that the opposing button had an auxiliary die.  Why would Guillermo not have the same problem?

Reply:
Oh, i see what you're saying from the original statement of the pull --- the rules are actually different for Gordo vs Guillermo.  Well, okay, i will update the dev site then.  Is there really no better way to identify buttons that work like Gordo, though?  Grepping by button name deep in the backend seems really clunky.

Reply:
Last night, I was thinking that it would be okay, it's only a once-off. However, thinking about it now, I think I agree that it's a bit clunky. I'll see if I can implement some sort of hooked function to move the logic to the BtnSkill classes.